designs/virtualization/4.5/Scheduling/Tolerations/tolerations.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# VM Tolerations
+
+Managing VM tolerations for tainted nodes
+
+![image name](img/Tolerations-0-0.jpg)
```

Comment:
just a nit but the heading should read "Scheduling and resource requirements" drop the plural on resource.